contrib/opencensus-samples/azure_monitor/src/azure_monitor/flask_sample/app/__init__.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019, OpenCensus Authors
```

Comment:
Maybe having the sample inside https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure/examples would be cleaner, having this other folder just make it harder to discover

Reply:
The point of the sample package is so that users can install it and have them install integration packages as needed for the end-to-end applications. There might be also sample applications in the future that are for other exporters, so a generic `opencensus-samples` folder should be used since it is not `azure` specific.